bin/morgue.js

```diff
@@ -5067,6 +5068,24 @@ function coronerNuke(argv, config) {
     errx('No such object.');
   }
 
+  if (!argv["-f"]) {
```

Comment:
Are we sure we want to provide this opt out from the verification?  We're going to break folks' command lines anyway with this change to prevent accidental foot gunning, but if we provide an opt out, then the command lines will simply adapt.

Reply:
The footgunning that this is reacting to was an interactive session, so it would prevent repeats of that.

I don't think removing the ability to script around `morgue nuke` is a sensible thing to do without more discussion